Resources/Prototypes/Reagents/biological.yml

```diff
@@ -1,10 +1,12 @@
-﻿- type: reagent
```

Comment:
sus

Reply:
What's wrong with it? When i compare it to other reagents, it looks normal
![image](https://github.com/space-wizards/space-station-14/assets/134914314/cc0d5712-ffbf-40d0-9692-fe3d7d40b832)

Reply:
the diff is sus

Reply:
ok i literally have no idea how to fix this but i doubt its what is causing such a big error